build-logic/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts

```
@@ -50,6 +50,9 @@ extraJavaModuleInfo {
     module("io.grpc:grpc-netty", "grpc.netty") {
         exportAllPackages()
         requireAllDefinedDependencies()
+        dependencies {
+            requires("java.logging")
+        }
```

```suggestion
        requires("java.logging")
```

Don't need the `dependencies {}` around this.